hazelcast-client/src/main/java/com/hazelcast/client/config/XmlClientConfigBuilder.java

```diff
@@ -92,6 +99,39 @@ public XmlClientConfigBuilder() {
         this.in = locator.getIn();
     }
 
+    @Override
+    protected Document parse(InputStream inputStream) throws Exception {
+
+        final DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+        Document doc;
+        try {
+            doc = builder.parse(inputStream);
+        } catch (final Exception e) {
+            String msg = "Failed to parse Config Stream"
+                    + "\nException: " + e.getMessage()
+                    + "\nHazelcastClient startup interrupted.";
+            LOGGER.severe(msg);
+            throw e;
+        } finally {
+            IOUtil.closeResource(inputStream);
+        }
+        return doc;
+    }
+
```

Comment:
No need to create a doc variable, assign to the doc variable and then return it.

Just do 'return builder.parse(inputStream)'. 

Saves you 2 lines of useless code :)